container_linux_test.go

```diff
@@ -1032,6 +1032,10 @@ func testUserNamespaces(t *testing.T, readonlyRootFS bool) {
 		return nil
 	})
 	if err != nil {
+		// message returned by runc, untyped
+		if msg := "USER namespaces aren't enabled in the kernel"; strings.Contains(err.Error(), msg) {
+			t.Skip(msg)
```

Comment:
Should we just run a quick program when starting the test and record the result in a boolean instead?

e.g.:
```go
	cmd := exec.Command("true")
	cmd.SysProcAttr = &syscall.SysProcAttr{
		Cloneflags: syscall.CLONE_NEWUSER,
	}

	err := cmd.Run()
```

Reply:
Let me try that

Reply:
That works fine, updated